compose-builder/Makefile

```
@@ -1035,7 +1038,7 @@ build-taf-nanomq:
 	make taf-compose-perf taf-perf-no-secty mqtt-bus nanomq no-secty
 
 compose: gen
-	cat gen-header docker-compose.yml > $(RELEASE_FOLDER)/docker-compose$(NO_SECURITY)$(APP_SAMPLE)$(BUS)$(NANOMQ)$(ARCH).yml
+	cat gen-header docker-compose.yml > $(RELEASE_FOLDER)/docker-compose$(NO_SECURITY)$(APP_SAMPLE)$(BUS)$(NANOMQ)$(ARCH)$(ZERO_TRUST_COMPOSE).yml
```

ARCH needs to always be last. 

```suggestion
	cat gen-header docker-compose.yml > $(RELEASE_FOLDER)/docker-compose$(NO_SECURITY)$(APP_SAMPLE)$(BUS)$(NANOMQ)$(ZERO_TRUST_COMPOSE)$(ARCH).yml
```